docusearch/templatetags/get_filename.py

```diff
@@ -4,5 +4,10 @@
 
 @library.global_function
 def get_filename(filepath):
-    """ Parses the filename from a document path """
-    return os.path.basename(filepath)
+    """
+    Parses the filename from a document path if no filepath present returns
+    `Document`
+    """
+    if filepath:
+        return os.path.basename(filepath)
+    return "Document"
```

Comment:
What about something like `No Document Title`, I just didn't want the default, default message to just say None.